workspace/__lib__/xod/debug/tweak-byte/patch.cpp

```diff
@@ -0,0 +1,10 @@
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    // The actual code that makes tweak-nodes work
+    // is automagically injected elsewhere
```

Comment:
```suggestion
    // is injected in detail::handleTweaks
```

...and it should be really there :)